WorkflowReactiveSwift/Sources/SignalWorker.swift

```diff
@@ -15,8 +15,16 @@
  */
 
 import ReactiveSwift
+import Workflow
+
+extension Signal: AnyWorkflowConvertible where Error == Never {
```

Comment:
Might be good to give some documentation here (and probably on SignalProducer too) with an example of calling `.running(in:)` directly on a signal.

Reply:
Added, please review.